### ISSUE TITLE
Do I need to poll to ensure that emitters created with `subscribeTo...` will emit anything?

### ISSUE THREAD

**Issue by andreashuber69** [alexbosworth/ln-service#177]
As far as I my tests indicate, the emitters created by at least some of the `subscribeTo...` functions will quietly stop working when LND restarts. So I guess I need to periodically monitor that LND is still alive and resubscribe when I detect that LND has restarted?

**Comment by alexbosworth** [comment]
None of them throw an error event?

**Comment by andreashuber69** [comment]
> None of them throw an error event?

You mean when calling subscribeToBlocks`](https://github.com/alexbosworth/ln-service#subscribetoblocks) while LND is down, for example? I did not test that. I trust that it would throw.

I only tested calling `subscribeToBlocks` when LND is still up. I then call `on` on the returned emitter and finally restart LND *after* subscribing. Since the docs do not mention a separate error event, I did not try to subscribe to that. After restarting LND I did not get any `"block"` events anymore. I guess that is as designed, I was just wondering whether there's an easier way to detect that I need to resubscribe?
